devel/api.md

```diff
@@ -885,6 +885,62 @@ Type: `integer`
 
 The Spanning Tree Protocol (STP) port cost for destinations via this port.
 
+## `LinuxBridge.Port.VLAN_SUBTREE`
+Each Linux Bridge port also features a subtree to configure the allowed
+VLANs passing through it. If the VLAN subtree is not configured, all the VLANs
+are allowed; once the subtree is defined, it will behave as a white-list.
+
+Example:
+
+```python
+{
+    LinuxBridge.Port.VLAN_SUBTREE: {
+        LinuxBridge.Port.Vlan.MODE: LinuxBridge.Port.Vlan.Mode.TRUNK,
+        LinuxBridge.Port.Vlan.ENABLE_NATIVE: True,
+        LinuxBridge.Port.Vlan.TAG: 105,
+        LinuxBridge.Port.Vlan.TRUNK_TAGS: [
+            LinuxBridge.Port.TrunkTags.ID: 100,
+            LinuxBridge.Port.TrunkTags.ID_RANGE: {
+                LinuxBridge.Port.TrunkTags.MIN_RANGE: 500,
+                LinuxBridge.Port.TrunkTags.MAX_RANGE: 599,
+            }
+        ]
+    }
+}
+```
+
+### `LinuxBridge.Port.VLAN.MODE`
+
+Type: `string`
+
+The operational mode of the port. Allowed values are:
+* trunk
+* access
```

Comment:
Should be `LinuxBridge.Port.Vlan.Mode.ACCESS` and `LinuxBridge.Port.Vlan.Mode.TRUNK`.

Reply:
Done